lib/ts/normalisedURLDomain.ts

```diff
@@ -32,7 +32,7 @@ function normaliseURLDomainOrThrowError(input: string, ignoreProtocol = false):
     input = input.trim().toLowerCase();
 
     try {
-        if (!input.startsWith("http://") && !input.startsWith("https://") && !input.startsWith("supertokens://")) {
+        if (!input.includes("://")) {
```

Comment:
Add a comment above this to explain why its done this way

Reply:
This comment makes no sense @iresharma. `which not the the desired result` what is the desired result that you are trying to achieve here?